src/templates/static/metatrader/download.jsx

```diff
@@ -1,64 +1,111 @@
-const DownloadBlock = ({
-    badge,
-    desc,
-    download,
-    href,
-    image,
-    os,
-    target,
-    title,
-}) => (
+const DownloadBlock = ({ badge, desc, download, href, image, os, target, title }) => (
     <div className='gr-4 gr-6-m gr-padding-10'>
         <img className='gr-6 gr-centered' src={it.url_for(`images/pages/metatrader/icons/${image}.svg`)} />
         <h3>{title}</h3>
-        <p>{desc }</p>
-        <a className={!badge ? 'button' : undefined} href={os ? `https://download.mql5.com/cdn/mobile/mt5/${os}?server=Binary.com-Server` : href } download={download || undefined} target={target || undefined}>
+        <p>{desc}</p>
+        <a
+            className={!badge ? 'button' : undefined}
+            href={os ? `https://download.mql5.com/cdn/mobile/mt5/${os}?server=Binary.com-Server` : href}
+            download={download || undefined}
+            target={target || undefined}
+        >
             <span className={badge || undefined}>{badge ? '' : it.L('Download')}</span>
         </a>
     </div>
 );
 
+const DesktopDownloadBlock = ({ badge, download, href, image, os, target, title, id }) => (
+    <div className='gr-12 gr-12-m gr-padding-10 invisible download-block' id={id}>
+        <img className='gr-6 gr-centered' src={it.url_for(`images/pages/metatrader/icons/${image}.svg`)} />
+        <a
+            className={!badge ? 'button' : undefined}
+            href={os ? `https://download.mql5.com/cdn/mobile/mt5/${os}?server=Binary.com-Server` : href}
+            download={download || undefined}
+            target={target || undefined}
+        >
+            <span className={badge || undefined}>{badge ? '' : it.L(`Download MT5 for ${title}`)}</span>
+        </a>
+    </div>
+);
+
 const Download = () => (
     <div id='mt_download' className='static_full center-text'>
-        <h1>{it.L('Start Trading with MetaTrader 5') }</h1>
-        <h3>{it.L('Trade with a powerful interface known as the global industry standard.') }</h3>
+        <h1>{it.L('Start Trading with MetaTrader 5')}</h1>
+        <h3>{it.L('Trade with a powerful interface known as the global industry standard.')}</h3>
 
         <div className='gr-padding-20'>
-            <h3>{it.L('For Desktop') }</h3>
-            <p>{it.L('Download MT5 for your desktop or laptop to access the powerful tools and features enjoyed by millions of traders.') }</p>
+            <h1 className='invisible download-heading' id='windows-heading'>
+                {it.L('MT5 for Windows')}
+            </h1>
+            <h1 className='invisible download-heading' id='macos-heading'>
+                {it.L('MT5 for Mac')}
+            </h1>
+            <h1 className='invisible download-heading' id='linux-heading'>
+                {it.L('MT5 for Linux')}
+            </h1>
+            <p>
+                {it.L(
+                    'Download MT5 for your desktop or laptop to access the powerful tools and features enjoyed by millions of traders.'
+                )}
```

Comment:
we don't really break localization strings into multiline, I think you can put it in one line instead of three

Reply:
This string is too long (more than 80 characters and reading it requires horizontal scrolling. Vertical texts are easier on the eye. 

@negar-binary Let me know if we have to do it in one-line only.

Reply:
it's more consistent with the rest of the code base, but it's up to you.